src/main/java/com/jalasoft/webservice/utils/FileManager.java

```diff
@@ -144,4 +144,14 @@ static public boolean isImageFile(String fullPathFile){
         return type.equals("image");
     }
 
+    /***
+     * Gets the file name with extension given a full file name eg. Tes1.csv
+     * @param fullFileName
+     * @return
+     */
+    static public String getFileName(String fullFileName) {
```

Comment:
what it is the fullfileName? is the file name + path?

Reply:
yes,it is